sonar-server/time-travel-server.go

```diff
@@ -220,8 +267,16 @@ func equivalentEvent(crucialEvent, currentEvent map[string]interface{}) bool {
 			} else {
 				return false
 			}
+
 		default:
-			log.Printf("Unsupported type: %v %T\n", v, v)
+			if e, ok := currentEvent[key]; ok {
+				if val == nil && e == nil {
+					log.Printf("Both nil type: %v and %v , key: %s\n", v, e, key)
```

Comment:
So did you encounter a case where both currentEvent[key] and crucialEvent[key] are of nil type?

Reply:
Yes, for example, `"lastProbeTime": null`, a filed is null in the json